src/components/IspSelect/react-select.scss

```diff
@@ -0,0 +1,371 @@
+/**
```

Comment:
I think this file should be put in `assets` since it is a vendor provided style that will be used probably by more than just this select component.